FiftyOne.Pipeline.Web.Shared/Services/ClientsidePropertyService.cs

```diff
@@ -240,14 +328,18 @@ private void SetHeaders(IContextAdapter context, string hash, int contentLength,
                 context.Response.SetHeader("Content-Length",
                     contentLength.ToString(CultureInfo.InvariantCulture));
                 context.Response.SetHeader("Cache-Control", _cacheControl);
```

Comment:
perhaps for the case of error it is worth to set `Cache-Control: no-cache` - otherwise javascript resource containing an error gets cached for 1800 seconds == 30 minutes - and that's how long the user would have to wait in case of the intermittent failure.